rfcs/067-prismic-api-ids/README.md

```diff
@@ -9,12 +9,19 @@
 We’d like to make sure our Prismic API IDs are written with consistent casing and that we’re following defaults from Prismic so that we’re going with the grain as far as possible and not having to manually override auto-generated SliceMachine files.
 
 ## Custom type API IDs
-These should be kebab-cased and plural when the type is ‘reusable’ as opposed to ‘single’ (e.g. the reusable ‘Exhibition highlight tour’ Custom type has the API ID `exhibition-highlight-tours`, whereas the single ‘Global alert’ Custom type has the API ID `global-alert`).
+When we started using Prismic we decided that these should be kebab-cased. Also, if the type was ‘reusable’ it should be plural, if it was 'single' it should be singular. For example, the reusable 'Exhibition highlight tour' type has an API ID of `exhibition-highlight-tours` whereas the single 'Global alert' type has an API ID of `global-alert`.
 
-SliceMachine makes these snake_case by default but allows this to be overridden easily in the GUI - we've done this fairly consistently although a couple of reusable content types have been given singular API IDs – `card` and `collection-venue`. I don't think there's any mileage in updating everything to snake_case since *none* of them are in that format currently. I'm not sure if we should try to update to `cards` and `collection-venues` or just live with those as they are?
+SliceMachine makes these snake_case by default but allows this to be overridden easily in the Graphical User Interface (GUI) and to-date we have made *all* of them kebab-case, so there probably isn't a good case for changing these to snake_case now.
```

Comment:
👍 
Let's consider this a documented decision and keep an eye should new content be created to ensure it's kebab-cased!

Reply:
Actually, quick thought, what about changing the decision to camelCase to reduce the amount of different casings we use?

Reply:
…or snake_case so that they're aligned with what we're going to (have to) go for with the Slice API IDs?

Reply:
Are these IDs used in our code/will cause TS pain?

Reply:
They aren't used as TS variables, no (they are currently kebab-case and that would already have been a problem for use in JS). They can only be used in quotes when querying Prismic.

Reply:
Right! So snake_case works too 👍